modules/ephemeral-storage-integration-between-shared-resources-insights-operator-and-openshift-builds.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: REFERENCE
+
+[id="ephemeral-storage-integration-between-shared-resources-insights-operator-and-openshift-builds_{context}"]
+= Integration between shared resources, Insights Operator, and OpenShift Builds
+
+[role="_abstract"]
+Integration between shared resources, Insights Operator, and OpenShift Builds makes using Red Hat subscriptions (RHEL entitlements) easier.
+
+Previously, in {product-title} 4.9.x and earlier, you manually imported your credentials or copied them to each project or namespace where you were running builds.
+
+Now, in {product-title} 4.10 and later, OpenShift Builds can employ Red Hat subscriptions (RHEL entitlements) by using shared resources and the simple content access feature provided by Insights Operator:
+
+* The simple content access feature imports your subscription credentials to a well-known `Secret` object.
+* The cluster administrator creates a `SharedSecret` custom resource (CR) around that `Secret` object and grants permission to particular projects or namespaces.
+* Builds that run within those projects or namespaces have access to the `SharedSecret` CR and its entitled RHEL content.
```

Comment:
```suggestion
* Builds that run within those projects or namespaces can mount a CSI Volume referencing the `SharedSecret` instance and its entitled RHEL content.
```

Reply:
Also, I would expect the example yaml for this this like we have in the blog post to be part of your updates (assuming a separate PR) to the existing Build article and using Red Hat subscriptoins.  But you will update this article at that time to link to that new content.

Reply:
Updated. Thanks. Keeping this comment open for now as a reminder.